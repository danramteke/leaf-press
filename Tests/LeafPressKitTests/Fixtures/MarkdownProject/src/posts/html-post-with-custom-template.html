published: August 1, 2020
template: custom.leaf
title: Custom Template HTML Post
---
<h1>Custom Template HTML Post</h1>

<p>HTML blog posts can have a custom template by setting <code>template: templateName.leaf</code> in the front matter.</p>
