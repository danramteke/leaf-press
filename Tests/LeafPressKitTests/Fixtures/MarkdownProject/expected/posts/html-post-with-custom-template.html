<html>
  <head>
    <link rel="stylesheet" href="/styles.css">
  </head>
  <body>
    <div>
    Custom template
    </div>
    <div id="custom-content">


<h1>Custom Template HTML Post</h1>

<p>HTML blog posts can have a custom template by setting <code>template: templateName.leaf</code> in the front matter.</p>



    </div>
  </body>
</html>
